### PR TITLE
Add battery percentage to features

### PR DESCRIPTION
**Pull request by norgeous**
~Early PR. WIP. do not merge~

**ready to go!**

Tested with:
```
wget https://github.com/norgeous/node-chip-io/archive/batterypercent.zip
unzip batterypercent.zip
cd node-chip-io-batterypercent/
sudo npm i
node examples/battery-percentage.js
```

Get battery percentage on the command line (for checking that it works):
```
echo $(($(i2cget -y -f 0 0x34 0xb9)))
```

### CODE REVIEW

**Comment by norgeous** [comment]
ok, its good! please review it @sandeepmistry

File: lib/axp209.js [hunk 1]
```diff
@@ -106,6 +119,16 @@ AXP209.prototype._readBatVolt = function(callback) {
...
 };
 
+AXP209.prototype._readBatPercent = function(callback) {
+  this._i2c.readByte(this._address, BAT_PERCENT_REGISTER, function(err, data) {
+    if (err) {
+      return callback(err);
+    }
+
+    callback(null, data);
+  }.bind(this));
```
Comment (fivdi) [inline lib/axp209.js]: Can
```js
this._i2c.readByte(this._address, BAT_PERCENT_REGISTER, function(err, data) {
  ...
}.bind(this));
```
be simplified to:
```js
this._i2c.readByte(this._address, BAT_PERCENT_REGISTER, callback);
```
?

File: lib/battery-percentage.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+var events = require('events');
+var util = require('util');
+
+var five = require('johnny-five');
+
+function BatteryPercentage(opts) {
+  this._sensor = new five.Sensor({
+    pin: 'BATP',
+    range: [0x00, 0xff]
```
Comment (sandeepmistry) [inline lib/battery-percentage.js]: I haven't had time to run this on my CHIP yet, but is the range of 0 - 255 correct? What range of values does the `BAT_PERCENT_REGISTER` contain?

**Comment by norgeous** [comment]
Yes, I think you found a error - the range is from 00 to 100.
With my battery reading 97%, the pin returns 0x61 raw.
I assume it should be `range: [0x00, 0x64]`.
Will adjust now.

**Comment by norgeous** [comment]
is there anything I can do to improve this? @sandeepmistry @fivdi

**Pull request closed without merging**